packages/graphql/lib/src/links/websocket_link/websocket_client.dart

```diff
@@ -166,7 +170,7 @@ class SocketClient {
 
```

Comment:
Replace `_connect` call in constructor with public `connect` call by invoker to allow blocking.

Reply:
I agree with this spec fix, but I'm not able why move the constructor to the public, can you add more reason for this choice?

Reply:
Fixed in https://github.com/maximilianmaihoefner/graphql-flutter/tree/69e46832999656979bf2c6829efc972087c01dff